infra_tooling_account/infra_tooling_account/grafana/glue_jobs_dashboard.sample.json

```diff
@@ -3,14 +3,11 @@
     "list": [
```

Comment:
According to these file's usage it might be better to call them .template.json, not .sample.json?

Reply:
yes, thank you :)